backstory/src/main/java/com/google/sps/servlets/data/UserServiceFactorySps.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets.data;
+
+import com.google.appengine.api.users.UserService;
+
+/**
+ * Factory pattern for UserService.
+ */
+public interface UserServiceFactorySps {
```

Comment:
Similar to above comment:
1. Can abbreviation be avoided,
2. Can Factory be the last word?

Reply:
Sure, I can change it so that factory is the last word, but since the SPS is what our folder is called I think it would be best to keep consistency in this way.

Reply:
Actually, I think I will call it BackstoryUserServiceFactory to avoid all of these problems.